guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,85 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the *Convert2RHEL* utility to simplify the conversion process.
+
+The *Convert2RHEL* utility in {Project} consists of an Ansible role and Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, enables required repositories and creates products, activation keys, and host groups.
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can use the Ansible role to generate conversion data for the following conversions:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
```

Comment:
@maximiliankolb When I mention other systems, then the problem is that the General Steps no longer apply and an alternative approach must be taken.